src/main/java/com/exasol/adapter/document/mapping/SourceReferenceColumnMapping.java

```diff
@@ -0,0 +1,60 @@
+package com.exasol.adapter.document.mapping;
+
+import com.exasol.adapter.metadata.DataType;
+
+/**
+ * This class describes a column that maps the source reference. For example in the files-virtual-schemas the name of
+ * the file that contained the document. The name of this column is always {@code SOURCE_REFERENCE}, except from if it
+ * is is used as foreign key (not yet possible).
+ */
+public final class SourceReferenceColumnMapping extends AbstractColumnMapping {
+    public static final String DEFAULT_COLUMN_NAME = "SOURCE_REFERENCE";
+    private static final long serialVersionUID = 9137288944756144081L;
+
+    /**
+     * Create an instance of {@link SourceReferenceColumnMapping}.
+     */
+    public SourceReferenceColumnMapping() {
+        super(DEFAULT_COLUMN_NAME);
+    }
+
+    private SourceReferenceColumnMapping(final String exasolColumnName) {
+        super(exasolColumnName);
+    }
+
+    @Override
+    public DataType getExasolDataType() {
+        return DataType.createVarChar(2000, DataType.ExaCharset.UTF8);
```

Comment:
Maybe add that is will be capped to 2000 characters in the user guide. Unlikely that paths exceeds it, but never know.